xml/admin_nfsganesha.xml

```diff
@@ -51,9 +47,29 @@
   <title>Installation</title>
 
   <para>
-   For installation instructions, see <!-- DEAD_LINK -->.
+   For installation instructions, see <xref linkend ="deploy-cephadm-day2-service-nfs"/>.
   </para>
  </sect1>
+
+ <sect1 xml:id="ceph-nfsganesha-namespace">
+   <title>Namespace Conventions</title>
+   <para>
+     The implementation conforms to Amazon Web Services (AWS) hierarchical
+     namespace conventions which map UNIX-style path names onto S3 buckets
+     and objects.
+   </para>
+   <para>
+     The top level of the attached namespace consists of S3 buckets,
+     represented as NFS directories. Files and directories subordinate to
+     buckets are each represented as objects, following S3 prefix and delimiter
+     conventions, with <literal>/</literal> being the only supported path delimiter.
+   </para>
+   <para>
+    S3 buckets are exported to NFS on a per-user basis, via this path convention:
+    <filename><replaceable>GANESHA_NODE:</replaceable>/<replaceable>USERNAME</replaceable>/<replaceable>BUCKETNAME</replaceable></filename>.
+   </para>
```

Comment:
```suggestion
```
Can also probably be removed .. S3 buckets are not exported by default and they are mounted using the explicitly defined pseudo path.

```
$ mount GANESHA_NODE:/pseudo-path /mnt
```